compiler/tfldump/src/OpPrinter.cpp

```diff
@@ -696,6 +710,7 @@ OpPrinterRegistry::OpPrinterRegistry()
   _op_map[tflite::BuiltinOperator_SUM] = make_unique<ReducerPrinter>();
   _op_map[tflite::BuiltinOperator_TRANSPOSE_CONV] = make_unique<TransposeConvPrinter>();
   // There is no Option for TOPK_V2
+  _op_map[circle::BuiltinOperator_UNIQUE] = make_unique<UniquePrinter>();
```

Comment:
ditto:)

Reply:
Sorry about the minor mistake, corrected it now. Thanks again.